### DISCUSSION TITLE
How to understand uct_iface_attr::latency?

### DISCUSSION THREAD

**Discussion by XCloudAI** [openucx/ucx#6979] [Q&A] [answered]
The docucment says ```Latency as function of number of active endpoints```.
What is measured for the ```latency``` here?

**Comment by yosefe** [comment, marked as answer]
Latency is time estimation (in seconds) for time spent in the transport device and network as function of number of active endpoints. Pls see usage example in https://github.com/openucx/ucx/blob/55c28859bc69c0b20520911e3af07611a1daf9a8/src/ucp/core/ucp_context.h#L437

**Comment by changchengx** [comment]
https://github.com/yosefe/ucx/blob/v1.9.0-pre38/src/uct/ib/base/ib_iface.c#L1303
https://github.com/yosefe/ucx/blob/v1.9.0-pre38/src/uct/ib/base/ib_iface.c#L1370
https://en.wikipedia.org/wiki/InfiniBand

**Comment by changchengx** [comment]
difference from overhead:
https://github.com/yosefe/ucx/blob/v1.9.0-pre38/src/uct/ib/rc/verbs/rc_verbs_iface.c#L183
